src/Ilios/CliBundle/Resources/config/services.yml

```diff
@@ -0,0 +1 @@
+services:
```

Comment:
Empty file.  Should it be removed?  Or it easier to keep it so adding things is less work?